vnext/Microsoft.ReactNative.Cxx/ReactMemberInfo.h

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+#ifndef MICROSOFT_REACTNATIVE_REACTMEMBERINFO
```

Comment:
Mentioned this on another instance of this, but do we need the header guard?

Reply:
Seeing this in a few more files as well

---
In reply to: [356920831](https://github.com/microsoft/react-native-windows/pull/3760#discussion_r356920831) [](ancestors = 356920831)

Reply:
See my answer in other place.

---
In reply to: [356921647](https://github.com/microsoft/react-native-windows/pull/3760#discussion_r356921647) [](ancestors = 356921647,356920831)